variables.tf

```diff
@@ -615,6 +615,12 @@ variable "gitlab_runner_registration_config" {
   }
 }
 
+variable "secure_parameter_store_registration_token_key" {
+  description = "The key name used store the Gitlab runner registration token in Secure Parameter Store"
```

Comment:
issue: I guess this is a copy & paste issue as the module does not store anything in the parameter. It uses the parameter to retrieve the GitLab registration token.

suggestion: `The name of the SSM parameter to read the GitlLb Runner registration token from.`